dask_glm/algorithms.py

```diff
@@ -304,7 +304,8 @@ def local_update(X, y, beta, z, u, rho, f, fprime, solver=fmin_l_bfgs_b):
 
 @normalize
 def lbfgs(X, y, regularizer=None, lamduh=1.0, max_iter=100, tol=1e-4,
-          family=Logistic, verbose=False, **kwargs):
+          family=Logistic, verbose=False, dask_distributed_client=None,
```

Comment:
Typically we would call this `client` if we're taking a Client or `scheduler_address`, if we're expecting a string address.

Reply:
We might also call `dask.distributed.get_client()` and rely on the global.  This is probably easier on users and unlikely to go wrong.  If we are in a situation where there are many clients (very rare) then the user could still control things with `dask.config.set(scheduler=client)`

Reply:
We could also use the `dask.distributed.get_client()` global for this.

```python
def compute_step(..., scatter=False):
    if scatter:
        from dask.distributed import get_client
        client = get_client()
        beta = client.scatter(beta)
    ...
```

This isn't as explicit, but I suspect that it will almost always be fine, and if someone has multiple clients (this is very rare) then they can control this with `dask.config.set(scheduler=client)`.

Reply:
I did the change using the dask.distributed method + add distributed in requirements